examples/twittermap/web/public/javascripts/map/countmap.js

```diff
@@ -113,11 +315,15 @@ angular.module('cloudberry.map')
           cloudberry.parameters.geoLevel);
         cloudberry.commonTimeSeriesResult = TimeSeriesCache.getTimeSeriesValues(cloudberry.parameters.geoIds,
           cloudberry.parameters.geoLevel, cloudberry.parameters.timeInterval);
+        $scope.ChartDataMap = TimeSeriesCache.getInViewTimeSeriesStore(cloudberry.parameters.geoIds,
+          cloudberry.parameters.timeInterval);
+        redrawPopup(1);
```

Comment:
Maybe it's more readable if argument using `true` instead of 1, and `false` instead of 0?